mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/SimulatorNode.java

```diff
@@ -3312,23 +3312,6 @@ public CommandResult runGenericMethodForCommand(CommandDescriptor c)
                 result.append(calcNMEAChecksum(result.toString()));
               }
             }
-          } else if (inputSentence.endsWith("GPGRS")) {
```

Comment:
@arturobernalg could you please explain to me why this disappears completely?

Reply:
HI @TanguySoto 
easy, it's duplicated. check 
https://github.com/esa/nanosat-mo-framework/blob/338f182e13749efb553dcea00a1011754dba96c4/mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/SimulatorNode.java#L3298